helm/cluster-gcp/values.yaml

```diff
@@ -27,7 +27,7 @@ controlPlane:
   replicas: 3  # Should be an odd number.
   rootVolumeSizeGB: 120
   serviceAccount:
-    email: "service-account@email"  # A service account used by the control-plane to set up LoadBalancer Services
+    email: "default"  # A service account used by the control-plane to set up LoadBalancer Services
```

Comment:
this is the default value that GCP uses.

Reply:
Can we add this change to the changelog too please?